CHANGELOG.md

```diff
@@ -79,6 +79,7 @@ New deprecation(s):
 
 ### Other
 
+- **General**: Add one deployment arg in keda-metrics-apiserver for only printing errors, but errors should go to stderr ([#4049](https://github.com/kedacore/keda/issues/4049))
```

Comment:
The message is a little bit confusing, could you please change it to something like you have in the PR name?

`- **General**: Show only logs with a severity level of ERROR or higher in the stderr ([#4049](https://github.com/kedacore/keda/issues/4049))`

Reply:
sure @zroubalik